lib/ui/process_ui/widgets/new_process_language_selection.dart

```diff
@@ -1,29 +1,193 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/src/widgets/framework.dart';
 import 'package:flutter/src/widgets/placeholder.dart';
+
+import 'package:provider/provider.dart';
 import 'package:registration_client/model/process.dart';
+import 'package:registration_client/provider/global_provider.dart';
+
 import 'package:registration_client/ui/process_ui/new_process.dart';
 
+import '../../../utils/app_config.dart';
+
 class NewProcessLanguageSelection extends StatelessWidget {
```

Comment:
This is not only for new process this will be applicable to all the registration processes. Lets name it as language_selector

Reply:
Done